x-pack/plugins/alerting/server/saved_objects/mappings.ts

```diff
@@ -56,6 +56,22 @@ export const alertMappings: SavedObjectsTypeMappingDefinition = {
           enabled: false,
           type: 'object',
         },
+        frequency: {
```

Comment:
If we're not querying these fields (`index: false`) why are we specifying these fields in the mappings?

Reply:
We use those fields but not query by them.
I can't create a rule that has those fields without adding them in the mapping, or am i wrong?

i get:
`mapping set to strict, dynamic introduction of [frequency] within [alert.actions] is not allowed: strict_dynamic_mapping_exception: [strict_dynamic_mapping_exception] Reason: mapping set to strict, dynamic introduction of [frequency] within [alert.actions] is not allowed`

if they are not in the mapping.